examples/gjs/adw-1-hello/package.json

```diff
@@ -20,11 +20,11 @@
   "author": "Pascal Garber <pascal@artandcode.studio>",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.24.0",
+    "@babel/core": "^7.24.5",
     "@rollup/plugin-babel": "^6.0.4",
-    "rollup": "^4.13.0",
-    "typescript": "^5.4.2",
-    "vite": "^5.1.6"
+    "rollup": "^4.17.2",
```

Comment:
In the future if we can separate version upgrades like this, it makes review faster

Reply:
okay